hbase-client/pom.xml

```diff
@@ -117,6 +117,12 @@
             <version>${testng.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Maybe add comments that explain that this needed because the minicluster dependencies don't pull this in in every version ?

Reply:
even though Omid itself uses testng

Reply:
Sure. Thanks.